doc/nrf/device_guides/working_with_nrf/nrf91/nrf9160.rst

```diff
@@ -263,6 +283,131 @@ Building and programming on the command line
 
       The device resets and runs the programmed sample or application.
 
+.. _nrf9160_ug_updating_fw_programmer:
+
+Updating the DK firmware using Programmer
+=========================================
+
+Complete the following steps to program applications using the Programmer app from `nRF Connect for Desktop`_.
+
+.. _nrf9160_updating_fw_modem:
+.. _nrf9160_gs_updating_fw_modem:
+
+Updating the modem firmware
+---------------------------
+
+To update the modem firmware, complete the following steps.
+If you experience any problems during the process, press ``Ctrl+R`` (``command+R`` on macOS) to restart the Programmer app and try again.
+
+.. note::
+
+   Updating the modem firmware erases the contents of the flash memory, so you must program the application again to the nRF9160 DK.
+
+1. Open the Programmer app.
+#. Make sure the **PROG/DEBUG SW10** switch on the nRF9160 DK is set to **nRF91**.
+   On DK v0.9.0 and earlier, this is the **SW5** switch.
+#. Connect the nRF9160 DK to the computer with a micro-USB cable, and then turn the DK on.
+#. Click :guilabel:`SELECT DEVICE` and select the DK from the drop-down list.
+   You can identify the nRF9160 DK by the fact that it has three COM ports.
```

Comment:
To see the COM ports, you have to hover over the button and click the small down arrow. I didn't even know that you could do that.

Isn't it easier to identify the DK by the fact that it says "nRF9160 DK"? 😄 Maybe this whole sentence could be just removed.

![image](https://github.com/nrfconnect/sdk-nrf/assets/2588548/fdea4b38-0773-4acf-aa59-6e7dff9306c7)

Reply:
That's exactly what I was thinking. It's probably better to use screenshots with the default view, because not all people (like me) even know that the device information can be expanded. 😄